scripts/core/editor3/helpers/editor3CustomData.js

```diff
@@ -0,0 +1,49 @@
+import {Map} from 'immutable';
+
+import {
+    SelectionState,
+    Modifier,
+    EditorState,
+} from 'draft-js';
+
+export const editor3DataKeys = {
+    MULTIPLE_HIGHLIGHTS_STORAGE_KEY: 'multiple-highlights',
+    RESOLVED_COMMENTS_HISTORY: 'resolved-comments'
+};
+
+export function keyValid(key) {
+    return Object.keys(editor3DataKeys).includes(key);
+}
+
+export function setCustomDataForEditor(editorState, key, value) {
+    if (keyValid(key)) {
```

Comment:
missing `!`

Reply:
Thanks for spotting this, I've noticed I've been validating it in the wrong way too! (fixed)